disnake/http.py

```diff
@@ -1175,6 +1177,83 @@ def get_thread_members(self, channel_id: Snowflake) -> Response[List[threads.Thr
         route = Route("GET", "/channels/{channel_id}/thread-members", channel_id=channel_id)
         return self.request(route)
 
+    def create_tag(
```

Comment:
The tag routes should have a warning that they're not finalized

Reply:
What about just deleting them for now? As well as tags types

Reply:
Yeah, since they're still wip that would make sense. Maybe keep the code around and add it later, once the API is stable 👍

Reply:
The only tag implementation is in typings and http.py, so I don't think we need to remove them as http.py is not a public facing api

Reply:
> The only tag implementation is in typings and http.py, so I don't think we need to remove them as http.py is not a public facing api

I am aware of this, but tags are "somewhat likely to change", so removing them for now is not a bad idea.  However, I want to keep a branch for specifically that, this PR will only contain basic and documented things

Reply:
Tags were removed in aa9055b7cbd5715b831383ce5112f98ed3623f20 😄